Examples/test-suite/cpp11_final_directors.i

```diff
@@ -0,0 +1,18 @@
+%module(directors="1") cpp11_final_directors
+
+%warnfilter(SWIGWARN_PARSE_KEYWORD) final;
```

Comment:
This should only be needed if something gets wrapped as the identifier `final` in the target language (e.g. if there was a method, class or variable called `final`) but that's not the case here.  That's because `final` is a keyword in some target languages (e.g. Java).

I tried removing this line and your new testcase still seems to pass.

Reply:
Thank you for your comment, @ojwb. You are absolutely right.

Also, this PR needs to add cpp11_final_directors to Examples/test-suite/common.mk.